module/platform/darwin/HardwareIO/src/HardwareIO.cpp

```diff
@@ -71,6 +71,43 @@ namespace platform {
             // Voltage (in volts)
             sensors.voltage = Convert::voltage(data.cm730.voltage);
 
+            if (sensors.voltage <= maxVoltage) {
+                sensors.cm730ErrorFlags &= ~DarwinSensors::Error::INPUT_VOLTAGE;
+            }
+
+            if (sensors.voltage <= minVoltage) {
+                servoTargetHandle.disable();
+
+                log<NUClear::FATAL>("CRITICALLY LOW VOLTAGE KILLING SERVOS");
```

Comment:
Fix the grammar in this sentence. Is the critically low voltage killing the servos? or are you killing the servos!